torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10639,7 +10639,10 @@ def error_inputs_mean(op_info, device, **kwargs):
                         # clamp_max supports two tensor input with bool, but not a bool scalar
                         DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_dtypes'),
                         # RuntimeError: "max_elementwise_cuda" not implemented for 'ComplexFloat'
-                        DecorateInfo(unittest.expectedFailure, 'TestBinaryUfuncs', 'test_type_promotion'),
+                        DecorateInfo(unittest.expectedFailure,
+                                     'TestBinaryUfuncs',
+                                     'test_type_promotion',
+                                     device_type='cuda'),
```

Comment:
Should this just xfail on the complex dtypes?

Reply:
Oh, no... that test isn't instantiated for multiple dtypes, I think. My mistake.